### ISSUE TITLE
hide a problematic OTU during OTU mapping workflow of the curator

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/opentree#956]
Sorting by "Unmapped OTUs first" is a really nice feature.

But if you have big study (lots of OTUs) and several are tips that are not in our taxonomy, they fail to map and then tend to clog the top of the screen. It would be nice to have a flag that would send them to the back of the list.

**Comment by jimallman** [comment]
This is a dupe of https://github.com/OpenTreeOfLife/opentree/issues/809, which has been fixed already. Does this address your concern?

(I see that there's a slight difference between **devtree** and **tree** (production), in that devtree  moves each OTU as it fails mapping, while production waits until we pause mapping and all failed OTUs move together. I'll chase this down in any case.)

**Comment by mtholder** [comment]
oh. sorry for the dupe. 
For lots of OTUs (and hence slow mapping) the devtree behavior seems better to me.

**Comment by mtholder** [comment]
Actually (after looking at https://tree.opentreeoflife.org/curator/study/edit/ot_700/?tab=otu-mapping again), I think this is an "almost duplicate."
There are some OTUs that are not in OTT. So I know that mapping them is hopeless. I'd like to have a way to force those to stay at the bottom of the list while I work on the others. Some of the others are cases in which I have just not crafted the correct regex for getting the mappable name from the full label. It is fine for those to fall to the back while I map, but I'd like to see them again as I work my way through the list.

This is probably a rare wish - so certainly not a high priority.

**Comment by jimallman** [comment]
> There are some OTUs that are not in OTT. So I know that mapping them is hopeless.

Thanks, I see the distinction now. Here's an idea for a Hide button (on the far right) that would remove this OTU entirely from the list, unless a special filter ("All hidden OTUs"?) is chosen. This would create an annotation on the OTU. Of course, there would be a corresponding Show button that will remove the annotation, restoring normal behavior and allowing mapping as usual.

![screen shot 2016-04-13 at 5 41 07 pm](https://cloud.githubusercontent.com/assets/446375/14510415/50ff6012-019f-11e6-8a76-506687bf2366.png)

**Comment by mtholder** [comment]
That sounds good to me. We could figure out a property name to add to the nexson (it could be a boolean within an otu, or one property that maps to a list of "hidden" otu ids).  
Or it could even be a non-persistent option that did not make it to the nexson, but was helpful during a single curation session.

**Comment by kcranston** [comment]
This is similar to #950 in that both issues request a way for a curator to assign a specific status to an OTU. In this case the tag is 'this is unmappable, so stop showing me this' while in #950 it is `this OTU should not be used for synthesis`.

**Comment by jimallman** [comment]
> This is similar to #950

Agreed, these seem similar to me. I'm wondering if the same UI (creates/removes an annotation with curator comment) would address both issues. If so, we could use the hide/show icons as shown above, or perhaps a "flag" icon and tell the UI to hide/show all flagged OTUs.
